src/tests/unit/tests/issue-filing/common/__snapshots__/issue-details-builder.test.ts.snap

```diff
@@ -19,3 +19,12 @@ h-How to fix-h--s-h-s--h-t--RR-failureSummary--h-t
 h-Environment-h--s-h-s--environmentName
 --f-s--This accessibility issue was found using app-name app-version (engine-name engine-version), a tool that helps find and fix accessibility issues. Get more information & download this tool at l-http://aka.ms/AccessibilityInsights-l."
 `;
+
+exports[`Name of the group builds issue details without snippet 1`] = `
+"h-Issue-h--s-h-s--s-RR-help-s (l-RR-help-url-RR-rule-id-l)
+h-Target application-h--s-h-s--pageTitle<x>
+h-Element path-h--s-h-s--RR-selector<x>
+h-How to fix-h--s-h-s--h-t--RR-failureSummary--h-t
+h-Environment-h--s-h-s--environmentName
+--f-s--This accessibility issue was found using app-name app-version (engine-name engine-version), a tool that helps find and fix accessibility issues. Get more information & download this tool at l-http://aka.ms/AccessibilityInsights-l."
```

Comment:
in the android report we decided to remove the engine version (axe-android version), I think we might need to remove it too from the issue filing, can we check with Fer?
![image](https://user-images.githubusercontent.com/45858632/82689847-fa89d180-9c0f-11ea-96d1-5e9b70eedaad.png)

Reply:
I will check with Fer. Do you think it would be reasonable to address this in a follow up PR if necessary?

Reply:
Yeah I'm fine addressing it in another PR